src/components/Sort.js

```diff
@@ -36,6 +38,42 @@ import {Spinner} from "react-bootstrap";
 
 const RowDivider = () => { return (<Row><Col>&nbsp;</Col></Row>); }
 
+export function getOktaModAccess(oktaGroups) {
```

Comment:
Did you merge main into your branch ?  We shouldn't need getOktaModAccess anymore, since we can get it from the store, and we should also get it from the store and compare it to testerMod to use the testerMod value

Reply:
@azurebrd @valearna which file have the most recent code as how to setup the access control so I can look at it ? BiblioFileManagement.js, BibliEditor.js, or Biblio.js  ?

Reply:
@azurebrd this PR not merge into main yet, so I can't use those for now.

Reply:
@FB-ping I'm pretty sure that PR is merged, it's got the Purple Merged label on the upper left.  Yeah, try what Valerio said, and merge main into your branch.  I linked the PR so you could see how I changed the code from the old way to the new way.